apps/src/code-studio/components/progress/UnitCalendarDialog.jsx

```diff
@@ -64,7 +84,7 @@ export default class UnitCalendarDialog extends Component {
         </div>
         <UnitCalendar
           lessons={lessons}
-          weeklyInstructionalMinutes={this.state.instructionalMinutes}
+          weeklyInstructionalMinutes={Number(this.state.instructionalMinutes)}
```

Comment:
This changes addresses a previously existing invalid prop type: 
![Screen Shot 2021-09-27 at 3 53 22 PM](https://user-images.githubusercontent.com/12300669/134976541-fc584f7d-1afb-4910-b73e-4fcd40009e42.png)

`instructionalMinutes` needs to be a number in `UnitCalendar` because it is used in re-sizing math.

Reply:
Great catch!  Should `this.state.instructionalMinutes` also be a number?  (It seems like this change should probably occur upstream of here somewhere?)

Reply:
+1 you could convert it to a number before setting it to state